### PR TITLE
Docs: pull docs from external Wrappers repo

### PR DESCRIPTION
**Pull request by burmecia**
## What kind of change does this PR introduce?

This PR is to pull latest markdown docs from https://github.com/supabase/wrappers and renders them in Supabase docs as part of the build pipeline.

This will pull docs from wrappers' main branch at build time. Anytime the Supabase docs site rebuilds, it will grab the latest docs from wrappers.

Currently pulls all the released foreign data wrapper docs:

- Airtable
- AWS S3
- BigQuery
- ClickHouse
- Firebase
- Logflare
- Stripe

Preview URL: https://docs-git-docs-external-wrappers-docs-supabase.vercel.app/docs/guides/database

## What is the current behavior?

Currently each foreign data wrapper's doc is linked externally to Wrappers' docs website.

## What is the new behavior?

All released foreign data wrapper's docs will be pulled from Wrappers' docs website and embedded in Supabase's docs website.

## Additional context

This work is to follow the [Federated docs](https://github.com/supabase/supabase/blob/master/DEVELOPERS.md#federated-docs) guidance.

### CODE REVIEW

**Comment by kiwicopple** [comment]
The preview for the Database section is here @burmecia :
https://docs-git-docs-external-wrappers-docs-supabase.vercel.app/docs/guides/database

Looks like you still need to add the sidebar links to `NavigationMenu.constants.ts`. Also, I would suggest creating an entirely new section (below "Extensions", don't lump them in with extensions"

The other thing that we need to fix is the "Tabs"

image

See how they work in Mkdocs here:

image

**Comment by gregnr** [comment]
@burmecia @kiwicopple Tabs plugin has been created (but not yet merged) in this PR:
https://github.com/supabase/supabase/pull/14861/files#diff-3ed447503759c3bfc25d2ec0ecaf394eb35a2a1ea1086dbfc0c509579d4a85d3

**Review by kiwicopple — APPROVED** [review]

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts [hunk 1]
```diff
@@ -660,6 +660,15 @@ export const database: NavMenuConstant = {
...
           url: '/guides/database/extensions/wrappers',
+          items: [
+            { name: 'Airtable', url: '/guides/database/extensions/wrappers/airtable' },
+            { name: 'AWS S3', url: '/guides/database/extensions/wrappers/s3' },
+            { name: 'BigQuery', url: '/guides/database/extensions/wrappers/bigquery' },
+            { name: 'ClickHouse', url: '/guides/database/extensions/wrappers/clickhouse' },
+            { name: 'Firebase', url: '/guides/database/extensions/wrappers/firebase' },
+            { name: 'Logflare', url: '/guides/database/extensions/wrappers/logflare' },
+            { name: 'Stripe', url: '/guides/database/extensions/wrappers/stripe' },
+          ],
```
Comment (kiwicopple) [inline apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:671]: let's move all these to a section below Extensions.

We can give the section the title "Wrappers"

Comment (burmecia) [reply apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts]: sure, done it.

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts [hunk 2]
```diff
@@ -657,10 +657,20 @@ export const database: NavMenuConstant = {
...
+      url: undefined,
+      items: [
+        { name: 'Overview', url: '/guides/database/extensions/wrappers/overview' },
+        { name: 'Airtable', url: '/guides/database/extensions/wrappers/airtable' },
+        { name: 'AWS S3', url: '/guides/database/extensions/wrappers/s3' },
+        { name: 'BigQuery', url: '/guides/database/extensions/wrappers/bigquery' },
+        { name: 'ClickHouse', url: '/guides/database/extensions/wrappers/clickhouse' },
+        { name: 'Firebase', url: '/guides/database/extensions/wrappers/firebase' },
+        { name: 'Logflare', url: '/guides/database/extensions/wrappers/logflare' },
+        { name: 'Stripe', url: '/guides/database/extensions/wrappers/stripe' },
```
Comment (kiwicopple) [inline apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:673]:
```suggestion
        { name: 'Overview', url: '/guides/database/extensions/wrappers/overview' },
        { name: 'Connecting to Airtable', url: '/guides/database/extensions/wrappers/airtable' },
        { name: 'Connecting to AWS S3', url: '/guides/database/extensions/wrappers/s3' },
        { name: 'Connecting to BigQuery', url: '/guides/database/extensions/wrappers/bigquery' },
        { name: 'Connecting to ClickHouse', url: '/guides/database/extensions/wrappers/clickhouse' },
        { name: 'Connecting to Firebase', url: '/guides/database/extensions/wrappers/firebase' },
        { name: 'Connecting to Logflare', url: '/guides/database/extensions/wrappers/logflare' },
        { name: 'Connecting to Stripe', url: '/guides/database/extensions/wrappers/stripe' },
```

File: apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts [hunk 3]
```diff
@@ -657,10 +657,20 @@ export const database: NavMenuConstant = {
...
+    },
+    {
+      name: 'Wrappers',
+      url: undefined,
+      items: [
+        { name: 'Overview', url: '/guides/database/extensions/wrappers/overview' },
+        { name: 'Connecting to Airtable', url: '/guides/database/extensions/wrappers/airtable' },
+        { name: 'Connecting to AWS S3', url: '/guides/database/extensions/wrappers/s3' },
+        { name: 'Connecting to BigQuery', url: '/guides/database/extensions/wrappers/bigquery' },
+        { name: 'Connecting to ClickHouse', url: '/guides/database/extensions/wrappers/clickhouse' },
```
Comment (kiwicopple) [inline apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts:670]:
```suggestion
        { 
          name: 'Connecting to ClickHouse', 
          url: '/guides/database/extensions/wrappers/clickhouse' 
        },
```

**Pull request merged**